tests/integration_tests/tests/agentless_tests/test_workflow.py

```diff
@@ -57,6 +57,27 @@ def test_execute_operation(self):
         outputs = self.client.deployments.outputs.get(deployment.id).outputs
         self.assertEquals(outputs['ip_address'], '')
 
+    def test_restart(self):
+        dsl_path = get_resource('dsl/basic.yaml')
+        blueprint_id = self.id()
+        deployment, _ = self.deploy_application(
+            dsl_path,
+            blueprint_id=blueprint_id,
+            timeout_seconds=15
+        )
+        execution = self.execute_workflow('restart', deployment.id)
+        events = self.client.events.list(
+            execution_id=execution.id, include_logs=True)
+        self.assertTrue(
+            any('stopping machine' in event['message'] for event in events))
+        self.assertTrue(
+            any('starting machine' in event['message'] for event in events))
+        data = self.get_plugin_data(
+            plugin_name='cloudmock',
+            deployment_id=deployment.id
+        )
+        self.assertEqual(list(data['machines'].values())[0], 'running')
```

Comment:
What does this test?

Reply:
yeah, nothing which isn't already tested by checking the logs